### PR TITLE
Completed html-forms exercise.

### PR DESCRIPTION
**Pull request by samlee0258**
Kapture 2022-12-06 at 11 58 40

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form>
+      <label>Email</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:9]: This `label` is not properly configured to pair with its form control element.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      </label>
+      <label>
+        <input type="radio" name="coding-language" value="javascript"> JavaScript
+      </label>
+      <p>Would you like to sign up for more information?</p>
+      <label>
+        <input type="checkbox" name="sign-up" value="yes"> Sign me up!
+      </label>
+      <div>
+        <label>Comments</label>
```
Comment (codingcodymiller) [inline html-forms/index.html:37]: This `label` is not properly configured to pair with its form control element.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: html-forms/index.html (hunk 2 above)
Comment (samlee0258) [reply html-forms/index.html]: Screenshot 2022-12-06 at 3 03 11 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
This is still not configured correctly. You should be able to click on the label to put focus on the form control element it is labeling.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**